### PR TITLE
Update and improve docs pt.1, plus misc fixes

### PR DESCRIPTION
**Pull request by aske**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,129 +5,162 @@
 [herc badge]: https://img.shields.io/badge/ci--by--hercules-green.svg
 [herc link]: https://hercules-ci.com/github/mlabs-haskell/plutip
 
-Plutip is Cardano tool for spawning local clusters.
-Use it to start up disposable private network with arbitrary amount of funded addresses (providing keys for that addresses as well). 
+Plutip is a Cardano tool for spawning local clusters.
+You can use it to start up disposable private network with arbitrary amount of funded addresses (providing keys for those addresses as well).
```
Comment (klntsky) [inline README.md:9]: Plutip generates these keys, not the user

File: README.md [hunk 2]
```diff
@@ -5,129 +5,162 @@
...
-Available options mostly match the `withFundedCluster` interface, see `--help` and [README](local-cluster/README.md).
 
-### Via CTL
+Available options mostly match the `withFundedCluster` interface, see `--help` and [local-cluster README](local-cluster/README.md) for detailed description of the arguments.
+
+### Via CTL for contract testing
 
-CTL is a purescript sdk for creating dapps. One of its features is ability to test contracts with private networks, see [plutip-testing](https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/doc/plutip-testing.md).
+[CTL](https://github.com/Plutonomicon/cardano-transaction-lib) is a PureScript SDK for creating DApps.
+One of its features is the ability to test contracts on disposable private networks which Plutip sets up, see [plutip-testing](https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/doc/plutip-testing.md).
```
Comment (klntsky) [inline README.md:141]: Please mention that users should not install plutip if the want to use it with ctl, because it provides everything needed

**Comment by zmrocze** [comment]
other than the two comments all the changes look correct

**Review by zmrocze — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -5,129 +5,165 @@
 [herc badge]: https://img.shields.io/badge/ci--by--hercules-green.svg
 [herc link]: https://hercules-ci.com/github/mlabs-haskell/plutip
 
-Plutip is Cardano tool for spawning local clusters.
-Use it to start up disposable private network with arbitrary amount of funded addresses (providing keys for that addresses as well). 
+Plutip is a Cardano tool for spawning local clusters.
+You can use it to start up disposable private network with an arbitrary amount of funded addresses (Plutip will provide corresponding PKHs and key pairs as well).
```
[the remark this answers is not in the document]
Comment (aske) [reply README.md:9]: Hm, feels like it doesn't give enough info this way

File: README.md [hunk 4]
```diff
@@ -5,129 +5,166 @@
...
```
Comment (klntsky) [inline README.md:9]:
```suggestion
You can use it to start up disposable private network with an arbitrary amount of funded addresses (Plutip will provide corresponding key pairs as well).
```

PKH is a part of the key pair

File: README.md [hunk 5]
```diff
@@ -5,129 +5,166 @@
...
 
-TLDR: `plutip :: IO CardanoNodeSocket` where the node belongs to a small cluster, optionally you can ask for prefunded keys.
+For smart contract testing see [CTL integration with Plutip](https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/doc/plutip-testing.md).
 
-## Requirements
+**TL;DR**: plutip gets you a cardano node socket where the node belongs to a small cluster on a private network, optionally you can prefund addresses with ADA.
```
Comment (klntsky) [inline README.md:13]:
```suggestion
**TL;DR**: plutip gets you a cardano node socket where the node belongs to a small cluster on a private network, and you can prefund addresses with ADA.
```

**Review by klntsky — CHANGES_REQUESTED** [review]

**Comment by klntsky** [comment]
All good
